faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
+        - Click on the green bug icon within one of the red error datasets to start the submission process 
+    - **Green unexpected-result datasets**:
```

Comment:
To be honest, I had to go back to the original FAQ to understand what it is about.
So maybe add a description about the problem encountered (task is sucessful e.g. green dataset but the result is not what you expect).

Reply:
Sure, I will look into it, and try to make it more understandable.